tests/system/providers/amazon/aws/example_dynamodb_to_s3.py

```diff
@@ -134,7 +134,8 @@ def delete_dynamodb_table(table_name: str):
         dynamodb_table_name=table_name,
         file_size=1000,
         s3_bucket_name=bucket_name,
-        export_time=datetime(year=2023, month=4, day=10),
+        export_time=datetime.now(),
```

Comment:
Is this change significant?

Reply:
Yes, because in system tests we just want to make sure the DynamoDBToS3Operator is successfully able to call `export_table_to_point_in_time` with the minimum time window. So the exported data is small. But I'm open to any suggestions.

Reply:
cc: @ferruzzi @o-nikolas @vincbeck

Reply:
Yes, I have tested it on local and I have been developing using `datetime.now()` only, no issues so far.